frontend/src/lib/components/PropertyKeyInfo.js

```diff
@@ -118,8 +118,8 @@ export const keyMapping = {
         description: '',
     },
     $anon_distinct_id: {
-        label: '$ce_version',
-        description: '',
+        label: 'Anon Distinct ID',
+        description: 'If the user ',
```

Comment:
looks like you didn't finish the description here

Reply:
Ha! Fixed